provision/acc_provision/templates/provision-config.yaml

```diff
@@ -59,6 +59,11 @@ net_config:
                                    # CNI should not wait for datapath to be
                                    # ready before returning success for pod
                                    # interface creation
+ #duration_wait_for_network: 210  # Duration in seconds that ACI should wait
```

Comment:
this example is not consistent with the value you are reading, suggestion is to change that, but either way it should be consistent